topology/probes/k8s/probe.go

```diff
@@ -95,3 +101,38 @@ func NewProbe(g *graph.Graph, manager string, subprobes map[string]Subprobe, lin
 		linkers:   linkers,
 	}
 }
+
+func logOnError(err error) {
+	logging.GetLogger().Warning(err)
+}
+
+type errorThrottle struct {
+	period   time.Duration
+	lastLock sync.Mutex
+	last     time.Time
+}
+
+func (r *errorThrottle) onError(error) {
+	r.lastLock.Lock()
+	defer r.lastLock.Unlock()
+	d := time.Since(r.last)
+	if d < r.period {
+		time.Sleep(r.period - d)
+	}
+	r.last = time.Now()
+}
+
+func muteInternalErrors() {
+	throttle := errorThrottle{
+		period: time.Duration(5) * time.Second,
```

Comment:
time.Duration(100) * time.Millisecond
would be enough here as it will generate only 10 lines/messages per second